### DISCUSSION TITLE
New Case

### DISCUSSION THREAD

**Discussion by mrijkhals** [BitMaker-hub/NerdMiner_v2#319] [General]
Hi guys,
I have made the V2 with the 3S board and decided to 3D print the case with fan for it.
Due to the 3D printer (resin) the push buttons were hard to push in so i made one with buttons. But the position of the case and fan was not really ideal. 
So i made a whole new case with integrated 30mm fan.
And another one in a small form without fan but with pushbuttons
Question: is there anyone that want to have it for their project?? If so, where can i post it.... link or stl files.

regards
Marc

**Comment by sany3001** [comment]
My favorite place: https://www.printables.com/
I will pick your work. :-)

**Reply by mrijkhals** [reply]
whoow there are alot of them on printables haha....  For now i have them stored free to download in tinkercad

**Comment by mrijkhals** [comment]
For now i have them stored free to download in tinkercad

**Reply by sany3001** [reply]
Will you drop link here? :-)

**Reply by mrijkhals** [reply]
no problem. But tinkercad haves a bug currently. Every project is staying private, even when i choose for public.
https://www.tinkercad.com/things/0ct6e06Rljp-nerd-miner-v2-case-small
https://www.tinkercad.com/things/2L80SBm5Ryu-nerd-miner-v2-desktop-case-w-internal-fan

**Comment by BitMaker-hub** [comment]
Thank you!
